.eslintrc.json

```
@@ -1,3 +1,4 @@
 {
-  "extends": "tamia"
+	"extends": "tamia",
```

```suggestion
	"extends": "tamia/react",
```

And you don't need to change the parser ;-)

---

Thanks, will do.